src/index.ts

```diff
@@ -1,5 +1,5 @@
 import * as core from "@actions/core";
-import fs from "fs-extra";
+import fs from "node:fs";
```

Comment:
Synchronous / Blocking FS calls should always be avoided, you can import the promises object which includes async fs calls and use it directly:

```suggestion
import {promises as fs} from "node:fs";
```

Reply:
ye, right - this action is really only executed alone though and its flow is really linear *anyway*, so using sync variants felt just simpler. I switched back everything to `fs/promises` now though